### ISSUE TITLE
Timeline: Accessibility issues

### ISSUE THREAD

**Issue by lv75** [SAP/ui5-webcomponents#3459]
**Describe the bug**
As shown in the screenshots, the React UI5 Timeline and TimelineItem components have missing or misplaced ARIA role attributes, which are applied to the shadow DOM elements and cannot be changed. But they do affect accessibility.

**Expected behavior**
Can you please check, and fix or provide the details how we can avoid these A11y issues.
Thanks.

**Screenshots**
![Screen Shot 2021-06-22 at 7 22 20 PM](https://user-images.githubusercontent.com/8533257/123027887-64d0f080-d393-11eb-81c6-5a8ab3b102a4.png)
![Screen Shot 2021-06-22 at 7 23 52 PM](https://user-images.githubusercontent.com/8533257/123027971-803bfb80-d393-11eb-9683-f11ce5b8b289.png)

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 0.31.9
`@ui5/webcomponents-react` version: 0.131.9
Operating System: MacOS
Browser: FF, Chrome et all.

**Comment by NHristov-sap** [comment]
Hi @lv75,

Can you provide an isolated sample in [https://codesandbox.io/](https://codesandbox.io/) that demonstrates the issue? Inspecting/debugging of screenshot is hard sometimes :)

Best Regards,
Nikolay Hristov

**Comment by gmkv** [comment]
Hi @lv75 

What is the tool for this testing that you're using?
For HTML/ARIA validation we recommended the testing tool [Access Assistant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm).
Can you also let me know what's the environment/storybook link that you're testing?
If it's something internal, you could reach out to me on Slack or Teams.

Regards,
Georgi Minkov

**Comment by lv75** [comment]
Hi @gmkv ,

Please see my reply on slack.

- Lucian

**Comment by gmkv** [comment]
Hello @SAP/ui5-webcomponents-topic-b 

Lucian has provided an isolated sample https://codesandbox.io/s/aged-resonance-imgm6 where you can run the [Access Assitant](https://chrome.google.com/webstore/detail/access-assistant/ojiighldhdmahfdnhfdebnpmlbiemdfm) tool to see for the reported ACC concerns.

Would you take a look?

Kind regards,
Georgi

**Comment by rchrdchn** [comment]
Hi guys - what's the estimated time to merge this fix to master and release it? Our team is waiting for this fix so we can close https://github.com/SAP/ui5-webcomponents/issues/3794 before the release this week.

**Comment by unazko** [comment]
Hello @rchrdchn,

This issue caused an internal discussion with accessibility expert involved.
I'll update you with an estimation as soon as possible.

Best Regards,
Boyan

**Comment by ilhan007** [comment]
A fix has been merged, you can expect a release on Monday. We will keep the issue open until you confirm it's resolved.

**Comment by ilhan007** [comment]
Hello @lv75 the issue is resolved with 0.31.20, could you try the new version and let us know if everything is fine.

**Comment by ilhan007** [comment]
No response, closing the issue.
